internal/seclang/directives.go

```diff
@@ -607,25 +618,37 @@ func directiveSecAuditLogRelevantStatus(options *DirectiveOptions) error {
 // Syntax: SecAuditLogParts [PARTLETTERS]
 // Default: ABCFHZ Note
 // ---
-// The format of the audit log format is documented in detail in the Audit Log Data Format Documentation.
+// The format of the audit log format is documented in detail in the Audit Log Data
+// Format Documentation.
+//
 // Example:
-// ```
+// ```apache
 // SecAuditLogParts ABCFHZ
 // ```
 //
 // Available audit log parts:
 //
 // A: Audit log header (mandatory).
 // B: Request headers.
-// C: Request body (present only if the request body exists and ModSecurity is configured to intercept it. This would require SecRequestBodyAccess to be set to on).
+// C: Request body (present only if the request body exists and ModSecurity is configured
```

Comment:
```suggestion
// C: Request body (present only if the request body exists and Coraza is configured
```
nit: some ModSecurity leftovers here and there